src/utils/data/vatglasses-helper.ts

```diff
@@ -0,0 +1,143 @@
+import { featureCollection } from '@turf/helpers';
+import { flattenEach } from '@turf/meta';
+import union from '@turf/union';
+import difference from '@turf/difference';
+import intersect from '@turf/intersect';
+import truncate from '@turf/truncate';
+import mergeRanges from 'merge-ranges';
+import type { Feature as TurfFeature, Polygon as TurfPolygon, MultiPolygon as TurfMultiPolygon } from 'geojson';
+
+
+export function splitSectors(sectors: TurfFeature<TurfPolygon>[]) {
+    let resultPolygons: TurfFeature<TurfPolygon>[] = [];
+
+
+    try {
+        for (let i = 0; i < sectors.length; i++) {
+            const currentPolygon = sectors[i];
+            truncate(currentPolygon, { mutate: true, precision: 1 });
+            let offset = 0;
+            if (currentPolygon.properties?.max % 10 !== 0 && currentPolygon.properties?.max % 10 !== 5 && currentPolygon.properties?.max !== 999) {
+                offset = 1;
+            }
+            // currentPolygon.properties.altrange = [[currentPolygon.properties?.min, currentPolygon.properties?.max % 10 === 4 ? currentPolygon.properties?.max + 1 : currentPolygon.properties?.max]];
+            if (currentPolygon.properties) currentPolygon.properties.altrange = [[currentPolygon.properties?.min, currentPolygon.properties?.max + offset]];
+            // We do the +1, because often the maximum is ending with 4, for sectors which go to FL315. But we need to have a matching min and max value for the mergeRanges to be working.
+            if (!resultPolygons.length) {
+                resultPolygons.push(currentPolygon);
+                continue;
+            }
+            const newResultPolygons: TurfFeature<TurfPolygon>[] = [];
+
+            let remainingOfCurrentPolygon: TurfFeature<TurfPolygon | TurfMultiPolygon> | null = currentPolygon;
+            for (const resultPolygon of resultPolygons) {
+                if (!remainingOfCurrentPolygon) {
+                    newResultPolygons.push(resultPolygon);
+                    continue;
+                }
+                const intersection = intersect(featureCollection([remainingOfCurrentPolygon, resultPolygon]));
+
+                if (intersection) {
+                    const difference1 = difference(featureCollection([remainingOfCurrentPolygon, resultPolygon]));
+                    const difference2 = difference(featureCollection([resultPolygon, remainingOfCurrentPolygon]));
+
+                    if (difference1) {
+                        difference1.properties = JSON.parse(JSON.stringify(difference1.properties));
```

Comment:
Let's change all of those to structuredClone. Should work fine on **backend** (also it's faster)